vendor/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -83,6 +83,7 @@ func NewGenericWebhook(scheme *runtime.Scheme, codecFactory serializer.CodecFact
 	clientConfig := rest.CopyConfig(config)
 
 	codec := codecFactory.LegacyCodec(groupVersions...)
+	clientConfig.ContentType = runtime.ContentTypeJSON
```

Comment:
isn't this change strange ?

Reply:
No, it's a commit from upstream that I am the author of.